modules/ebi-metagenomics/foldcomp/compress/main.nf

```diff
@@ -0,0 +1,49 @@
+process FOLDCOMP_COMPRESS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/foldcomp:0.0.5--h43eeafb_2':
+        'biocontainers/foldcomp:0.0.5--h43eeafb_2' }"
+
+    input:
+    tuple val(meta), path(pdb)
+
+    output:
+    tuple val(meta), path("{${meta.id}_fcz,*.fcz}"), emit: fcz
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def ext = pdb.isDirectory() ? "_fcz" : ".fcz"
+    """
+    foldcomp \\
+        compress \\
+        -t ${task.cpus} \\
+        ${pdb} \\
+        ${prefix}${ext} \\
+        ${args}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        foldcomp: v0.0.5
```

Comment:
It shouldn't have the "v0.0.5", it should be "0.0.5"... foldcomp doesn't print the version (foldcomp --verison)?